packages/mesos-modules/windows.build.ps1

```diff
@@ -0,0 +1,114 @@
+$ErrorActionPreference = "stop"
+
+$PKG_DIR = "c:\pkg"
+
+function Install-OpenSSL {
```

Comment:
Installing OpenSSL and Mesos seems to repeat a lot of the work done in the `mesos` module. Is there a way to avoid this?

Do we need OpenSSL here, for Open DC/OS?

Reply:
yes, we do need. no, we cannot avoid at the moment. there was a separate discussion before around this since it impacts build time significantly.

Reply:
@jongiddy Yes; specifically, we are considering two options to get rid of redundant Mesos builds in the future:
1) Adding a package `mesos-build` that will be the only one that will actually build Mesos, will have all the Mesos build artifacts as an output, will be a dependency of `mesos`, `mesos-modules` and `dcos-ee-mesos-modules`, and will not be packaged into a final DCOS build. The last part needs support in pkgpanda.

 2) Producing a full-fledged installable package with all Mesos libraries and includes as a result of Mesos build. Needs quite some work in Mesos cmake build. Will make the DCOS build slightly heavier than (1)

Looks like neither of two can be done quickly enough, hence we are implementing this as an intermediate step.

Reply:
Do we have a ticket to track this? If not plese crate one and add TODO here.
I'm fine with merging not ideal but working solution

> Done is better than perfect